internal/telemetry/event_test.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package telemetry
+
+import (
+	"errors"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mongodb/mongocli/internal/config"
```

Comment:
run go import

Reply:
If this is now a requirement we should have a test for it.

Reply:
I see what you mean, turns out `goimports` respects manually grouped imports, we should manually remove all empty lines and then run `goimports`